test/distributed/fsdp/test_fsdp_checkpoint.py

```diff
@@ -115,7 +115,7 @@ def test_checkpoint_fsdp_wrapping(self, cpu_offload, offload_activations):
 
         models = [ckpt_sequential_wrapped_fsdp, inner_ckpt, baseline]
 
-        offload_to_cpu_event = "Memcpy DtoH"
+        offload_to_cpu_event = "Memcpy DtoH" if torch.version.cuda else "CopyDeviceToHost"
```

Comment:
No, this test is already tested just for cuda, otherwise offload from cpu doesn't make sense.

This check is to distinguish between CUDA and ROCm which report different profiling text for a DtoH copy.  Alternatively, could have been written as:

```suggestion
        offload_to_cpu_event = "CopyDeviceToHost" if torch.version.hip else "Memcpy DtoH"
```

but I thought it best to add the ROCm case to the end rather than reordering the line.

Reply:
@kumpera please let me know which of the two options you prefer above so we can move this PR towards closure.

Reply:
Oh! I wasn't aware that `torch.cuda.is_available()` returns true in both case. In this case, this looks good as is